pypesto/objective/priors.py

```diff
@@ -225,6 +229,68 @@ def residual_jacobian(self, x):
 
         return sres
 
+    def sample(
+        self, n_samples: int = 1, seed: int = None
+    ) -> dict[str, np.ndarray]:
+        """
+        Sample from the prior distribution. Parameters are returned on defined parameter scale.
+
+        Parameters
+        ----------
+        n_samples:
+            Number of samples to draw.
+        seed:
+            Random seed for reproducibility.
+
+        Returns
+        -------
+        samples:
+            Dictionary of samples from the prior distributions for each
+            parameter index.
+        """
+        rng = np.random.default_rng(seed)
```

Comment:
```suggestion
    def sample(
        self, n_samples: int = 1, rng: int = None
    ) -> dict[str, np.ndarray]:
        """
        Sample from the prior distribution. Parameters are returned on defined parameter scale.

        Parameters
        ----------
        n_samples:
            Number of samples to draw.
        rng:
            Random number generator or seed for reproducibility.

        Returns
        -------
        samples:
            Dictionary of samples from the prior distributions for each
            parameter index.
        """
        rng = np.random.default_rng(rng)
```

Let's stick to `rng`, this becoming kind of standard across various libraries.

Reply:
Then the typehint `rng: int` should also be adapted.